tests/pds/dataservice_test.go

```diff
@@ -117,10 +117,13 @@ var _ = Describe("{ValidatePDSHealthInCaseOfFailures}", func() {
 
 				log.Infof("PDS DataService Pods")
 				log.Infof("deployment name %v", *deployment.ClusterResourceName)
+				if len(pdsPods) != 0 {
+					pdsPods = nil
+				}
```

Comment:
Why are we doing this if we know it is empty? We don't need this

Reply:
`pdsPods` is a global variable and its used by other tests also, in some cases it will not be empty hence Im making it empty at start of the test